proxy stats matcher (#2178)
kyma-project/istio: docs/user/04-00-istio-custom-resource.md

| --- | --- | --- |
| **resources** <br /> [Resources](#resources) | Defines Kubernetes resources' configuration. See [Resource Management for Pods and Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/). | Optional |

### ProxyStatsMatcher

Configures the stats matcher for Istio proxy sidecars and gateways.

Appears in:
- [Config](#config)

| Field | Description | Validation |
| --- | --- | --- |

> | **inclusionRegexps** <br /> string array | Defines a list of regular expressions used to select proxy stats for inclusion.<br />Stats whose names match any of these regular expressions are included in the proxy stats output. | Optional <br /> |

Reviewer: ```suggestion
| **inclusionRegexps** <br /> string array | Defines a list of regular expressions used to select proxy statistics for inclusion.<br />Statistics with names that match any of these regular expressions are included in the proxy statistics output. | Optional <br /> |
```
